### ISSUE TITLE
v1.3.0 under NC28 - error "Page not found"

### ISSUE THREAD

**Issue by zenlord** [kirstenhh/nc-bpm-app#8]
Hi,
First off: thank you so much for the update!

I can confirm the automated update went flawlessly, but I cannot open existing BPMN files or create new BPMN files.

Whenever I click an existing file, I am greeted with the "Page not found error"
![image](https://github.com/kirstenhh/nc-bpm-app/assets/594242/e4907438-e425-400e-971e-34ce9ed0a17d)

The URL is 

    https://<domain>/apps/files_bpm/?dir=/Professional/Business%20Process%20Designs&fileId=234529"

The same error is shown when I try to create a new BPMN file. The URL in that case is

    https://<domain>/apps/files_bpm/?dir=/&ext=bpmn

I am not running the previewer service and I have left that configuration empty. The logs are not showing any errors or warnings (and info messages do not relate to the files_bpm app).

**Comment by kirstenhh** [comment]
Two things: 
- I can't replicate the error, can you send me a list of what other apps you've got running on the files page? (I figure I might be missing some exclusion, or using an API the wrong way.)
- Can you clear your cache? I kept breaking it in development when I forgot to clear it out.

**Comment by zenlord** [comment]
I have cleared Chrome's browser cache, but still the problem is the same. FYI this is on Windows, but can confirm same behavior on Linux+Firefox

This is the list of apps installed and enabled/disabled on my server:

````
occ app:list
Enabled:
  - activity: 2.20.0
  - bruteforcesettings: 2.8.0
  - calendar: 4.6.5
  - circles: 28.0.0-dev
  - cloud_federation_api: 1.11.0
  - comments: 1.18.0
  - contacts: 5.5.2
  - contactsinteraction: 1.9.0
  - cospend: 1.6.0
  - dashboard: 7.8.0
  - dav: 1.29.1
  - deck: 1.12.2
  - federatedfilesharing: 1.18.0
  - federation: 1.18.0
  - files: 2.0.0
  - files_automatedtagging: 1.18.0
  - files_bpm: 1.3.0
  - files_pdfviewer: 2.9.0
  - files_reminders: 1.1.0
  - files_sharing: 1.20.0
  - files_trashbin: 1.18.0
  - files_versions: 1.21.0
  - firstrunwizard: 2.17.0
  - flow_notifications: 1.8.0
  - logreader: 2.13.0
  - lookup_server_connector: 1.16.0
  - nextcloud_announcements: 1.17.0
  - notes: 4.9.2
  - notifications: 2.16.0
  - notify_push: 0.6.9
  - oauth2: 1.16.3
  - password_policy: 1.18.0
  - passwords: 2024.2.20
  - photos: 2.4.0
  - previewgenerator: 5.4.0
  - privacy: 1.12.0
  - provisioning_api: 1.18.0
  - recommendations: 2.0.0
  - related_resources: 1.3.0
  - richdocumentscode: 23.5.705
  - serverinfo: 1.18.0
  - settings: 1.10.1
  - sharebymail: 1.18.0
  - support: 1.11.0
  - survey_client: 1.16.0
  - systemtags: 1.18.0
  - tables: 0.6.6
  - tasks: 0.15.0
  - text: 3.9.1
  - theming: 2.3.0
  - theming_customcss: 1.15.0
  - twofactor_backupcodes: 1.17.0
  - updatenotification: 1.18.0
  - user_ldap: 1.19.0
  - user_status: 1.8.1
  - viewer: 2.2.0
  - weather_status: 1.8.0
  - workflow_pdf_converter: 1.13.0
  - workflow_script: 1.13.0
  - workflowengine: 2.10.0
Disabled:
  - admin_audit: 1.18.0
  - appointments: 1.15.5 (installed 1.15.5)
  - encryption: 2.16.0
  - extract: 1.3.6 (installed 1.3.6)
  - files_external: 1.20.0
  - files_rightclick: 0.15.1 (installed 1.6.0)
  - files_texteditor: 2.15.1 (installed 2.15.1)
  - fileslibreofficeedit: 1.1.0
  - metadata: 0.19.0 (installed 0.19.0)
  - suspicious_login: 6.0.0
  - twofactor_totp: 10.0.0-beta.2
  - user_pwauth: 2.7.4
````

**Comment by kirstenhh** [comment]
Unfortunately I still can't replicate your error, and I've got to move on to some other tasks for the moment so I won't have time to work on this for a bit. Can you try uninstalling the app completely and reinstalling, see if that helps anything?

**Comment by zenlord** [comment]
Apologies, @kirstenhh - the error appears to be on my side. I have updated NC from 28.0.2 to 28.0.3, but that did not help. Then my websever crashed and I restarted both Apache and PHP-FPM and suddenly bpm_files is working again. I could have sworn that I had restarted at least php-fpm and Apache before filing this bug, but may have forgotten to do so. So I guess some caching mechanism was still active...
